### PR TITLE
Update CloudEvents

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@microsoft.com>

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [x] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [x] Have you picked the single best (existing) category for your project?
* [x] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [x] Have you added your SVG to `hosted_logos` and referenced it there?
* [x] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] Does your project/product name match the text on the logo?
* [x] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [x] ~30 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

**Comment by duglin** [comment]
I can't view the logs from netlify to see what's wrong

File: landscape.yml [hunk 1]
```diff
@@ -3790,6 +3790,17 @@ landscape:
               artwork_url: https://github.com/cncf/artwork/blob/master/examples/sandbox.md#cloudevents-logos
               stack_overflow_url: https://stackoverflow.com/questions/tagged/cloudevents
               slack_url: https://cloud-native.slack.com/messages/cloudevents
+              summary_personas: ''
+              summary_tags: events,eventing,serverless
+              summary_use_case: Standardizing common eventing metadata and their location to help with event identification and routing.
+              summary_business_use_case: ''
+              summary_release_rate: 2019/10/24
```
Comment (onlydole) [inline landscape.yml:3797]:
```suggestion
              summary_release_rate: 2019-10-24
```

File: landscape.yml [hunk 2]
```diff
@@ -3790,6 +3790,17 @@ landscape:
...
+              summary_integration: ''
+              summary_intro_url: ''
+            project_org: https://github.com/cloudevents
+            description: Standardizing common eventing metadata and their location to help with event identification and routing.
+            url_for_bestpractices: https://bestpractices.coreinfrastructure.org/en/projects/6770
+            joined: 2018/05/15
```
Comment (onlydole) [inline landscape.yml:3803]:
```suggestion
            joined: 2018-05-15
```

**Review by onlydole — CHANGES_REQUESTED** [review]
Thank you for this PR, @duglin - I left some minor comments for updating the dates to a valid format for the landscape YAML.

File: landscape.yml (hunk 1 above)
Comment (duglin) [reply landscape.yml]: done

File: landscape.yml (hunk 2 above)
Comment (duglin) [reply landscape.yml]: done

**Comment by duglin** [comment]
@onlydole made the suggested edits. Any idea why netlify isn't happy?

**Comment by onlydole** [comment]
Looks like Netlify has an error and no logs are showing for me. 

@AndreyKozlov1984 - do you see what the error is?

**Comment by duglin** [comment]
LGTM

**Pull request merged**